frontend/src/shared/ui/CustomDropdownMenu/CustomDropdownMenu.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  InputLabel,
+  MenuItem,
+  Select,
+  type SelectChangeEvent,
+} from "@mui/material";
+import { useState, type FC } from "react";
+import { CustomFormControl } from "./CustomDropdownMenu.styles";
+
+interface Label {
+  first: string;
+  second: string;
+}
+
+const menuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: "360px",
+      width: "36px",
+    },
+  },
+};
+
+type DropdownMenuProps = {
+  width: string;
+  Labels: Label;
+  filteringValues: number[];
+};
+
+export const CustomDropdownMenu: FC<DropdownMenuProps> = (props) => {
```

Comment:
```suggestion
export const CustomDropdownMenu: FC<DropdownMenuProps> = ({
  width, labels, filteringValues
}) => {
```

Reply:
все еще актуально 
вариант 
```suggestion
export const CustomDropdownMenu: FC<DropdownMenuProps> = ({ width, labels, filteringValues }) => {
```
читается лучше, чем 
`  const { width, Labels, filterValues } = props;`